src/cli/config.js

```diff
@@ -92,7 +91,7 @@ export const loadSubConfigFile = (configFile: string): SubConfig => {
             );
         }
     });
-    validateOptions(configFile, data.options ?? {});
+    validateOrThrow(data, subSchema);
```

Comment:
I don't understand how `extends` works.  It looks like we're reading the sub config, but not copying any fields from the base config.  Does that happen somewhere else?

Reply:
yeah so this stuff is all getting replaced in https://github.com/Khan/graphql-flow/pull/48
there will only be a single config file, no `extends`.